src/test/resources/pages/InstructorHomeHTML.html

```diff
@@ -213,155 +237,173 @@ <h5 class="pull-left">
             <tbody>
                <tr id="session0">
                   <td>
-                     Second Feedback Session
+                     First Feedback Session
```

Comment:
I verified this on master. Without any other change, just changing the start and end times in InstructorHomePageUiTest3.json causes the default order of the sessions on this page to change.